cpp/src/slice2cs/CsUtil.h

```diff
@@ -127,8 +135,9 @@ class CsGenerator : private ::IceUtil::noncopyable
     std::string sequenceMarshalCode(
         const SequencePtr& seq,
         const std::string& scope,
-        const std::string& param,
-        bool forNestedType = false);
+        const std::string& v,
```

Comment:
It's not clear what `v` is here.

Reply:
It's a parameter or a field or a sequence element inside a sequence that it is itself a parameter / field (can also be deeper, or be inside a dictionary value).

I renamed `param` to `v` as param sounds like it can only be a parameter, which is odd with the addition of the `readOnlyParam` parameter which is true when `v` represents a read-only parameter. 

 You would prefer `value` or `seqValue` ?

Reply:
Or "name" if it's the name of the parameter/field? I don't know well this code so you know better which one to pick ;-).

Reply:
Switched to `value`.

Reply:
Switched to value.